redash/handlers/authentication.py

```diff
@@ -102,6 +122,14 @@ def forgot_password(org_slug=None):
     return render_template("forgot.html", submitted=submitted)
 
 
+@routes.route(org_scoped_rule('/send_verification'))
```

Comment:
Need to limit this only to a POST request.

Reply:
Already done earlier, probably didn't push yet.